app/src/main/res/layout/notification_cron_item.xml

```diff
@@ -40,6 +40,16 @@
         android:layout_toStartOf="@+id/notificationCronOptionsButton"
         android:textAppearance="@style/ListItemSecondaryTextAppearance" />
 
+    <TextView
```

Comment:
The marginBottom of the above element must be removed

Reply:
Done.